desktop-app/app/service/browserSync.js

```diff
@@ -0,0 +1,14 @@
+import {ipcRenderer} from 'electron';
+import browserSync from 'browser-sync';
+
+let browserSyncOptions;
+
+if (!browserSyncOptions) {
+  browserSyncOptions = ipcRenderer.sendSync('request-browser-sync');
```

Comment:
@esprush Can you convert this to `ipcRenderer.invoke` instead of `sendSync`? 
This is going to block the UI thread until the request is processed which is not right.